functions/exercises.py

```diff
@@ -240,7 +316,12 @@ def check_palindrome(string):
     :return:
     """
     # INSERT YOUR CODE HERE
-    #
+    string = string.replace(" ","")
+    rev = string [::-1]
+    if rev == string:
+        return True
+    else: 
+        return False

```

Comment:
```suggestion
    return rev == string:
```

the comparisson already returns a bool, no need to use if's